### ISSUE TITLE
Can we correct the mask in intermediate frames?

### ISSUE THREAD

**Issue by vedantroy** [gaomingqi/Track-Anything#112]
README states:
> During tracking, users can flexibly change the objects they wanna track or correct the region of interest if there are any ambiguities.

Does this mean it is possible to fix the mask in some intermediate frame? (E.g, if the mask flickers), and if so, how can I do it?

**Comment by baoyi-A** [comment]
Same question

**Comment by G-nn-r** [comment]
Yes, this is possible, but requires some manual work. You basically need to re-mask and re-track the affected frames, see here: https://github.com/gaomingqi/Track-Anything/blob/master/doc/tutorials.md#4-correction
